config/environments/local/local.node.config.toml

```diff
@@ -34,6 +34,7 @@ PollMinAllowedGasPriceInterval = "15s"
 
 [Etherman]
 URL = "http://your.L1node.url"
+ForkIDChunkSize = 2000
```

Comment:
I think we miss to define the default value for ForkIDCheckSize in default.go. 2000 is a good default value? or maybe we can use a higher value to reduce the amount of L1 block requests?

Reply:
WOOOW You are right. I forgot to add it there. Good catch!